charts/common/kubeflow-issuer/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v1.5.0"
```

Comment:
1.5?

Reply:
since cert-manager is v1.5.0

Reply:
got it, this is kubeflow specific so can do 1.6.1 here

Reply:
done